docs/developer-guide/debugging.md

```diff
@@ -95,55 +193,29 @@ mv start_openocd_f1 /usr/local/bin
 mv start_openocd_f4 /usr/local/bin
 ```
 
-### For VSCode
-
-For VSCode, install the openocd version currently available through your package manager:
-
-Ubuntu:
-```bash
-sudo apt install openocd
-```
-
-Mac:
-```bash
-brew install open-ocd
-```
-
-The `start_openocd_f4` script requires a few additional parameters to ensure proper connection to VSCode and GDB:
-
-```bash
-#!/bin/bash
-cd /opt/openocd/0.10.0-11-20190118-1134/bin
-./openocd -f interface/stlink.cfg -f target/stm32f4x.cfg -c "gdb_port 50250" -c init -c "reset init"
-```
-
-As shown above, this script can be added to your `/usr/local/bin` if you want to be able to call it from anywhere.
+### Install 32-bit Dependencies
 
-## Install ARM compiler and 32-bit Dependencies
-
-Follow the guide in [Building and Flashing](/developer-guide/building-flashing) to install the compiler.
-
-QtCreator also needs 32-bit python bindings to run GDB (skip this if using VSCode)
+QtCreator needs 32-bit python bindings to run GDB. Install these with
 
 ``` bash
 sudo dpkg --add-architecture i386
 sudo apt update
 sudo apt install libpython2.7:i386
 ```
 
-## Configure QtCreator for ARM Development
+### Configure QtCreator for ARM Development
 
 Open up the new QtCreator you just installed (make sure it's the new one, and not the version you get from `apt`)
```

Comment:
remove note about `apt` since 18.04 `apt` works

Reply:
Should remove the instructions above this for installing the downloaded version on 16.04 then? I assume we still need the 32-bit dependencies though for 18.04?